Remotion/Web/Development.WebTesting/WebTestHelper.cs

```diff
@@ -155,7 +155,7 @@ public ITestInfrastructureConfiguration TestInfrastructureConfiguration
     /// <summary>
     /// Coypu main browser session for the web test.
     /// </summary>
-    public IBrowserSession MainBrowserSession
+    public IBrowserSession? MainBrowserSession
```

Comment:
Considering OnFixtureSetup should be called first, we should make this not nullable and add the guard here. Then we can use the proprty instead of the field for accessing MainBrowserSession as not nullable.